persist/sqlite/metrics.go

```diff
@@ -66,7 +66,9 @@ func (s *Store) HostMetrics() (result explorer.HostMetrics, err error) {
 			}
 
 			result.TotalStorage += host.Settings.TotalStorage
+			result.TotalStorage += host.V2Settings.TotalStorage
 			result.RemainingStorage += host.Settings.RemainingStorage
+			result.RemainingStorage += host.V2Settings.RemainingStorage
```

Comment:
Make sure to update any tests to check those additional fields and also API docs.
Especially since `RemainingStorage` uses different units for v1 and v2. The latter uses sectors afaik. Would also double check `TotalStorage`.